lib/controller/StudyRequestController.js

```diff
@@ -233,7 +238,47 @@ StudyRequestController.push({
       }
     }
 
-    return StudyRequestDAO.update(studyRequestNew, user);
+    const tasks = [StudyRequestDAO.update(studyRequestNew, user)];
+    if (studyRequestNew.status !== studyRequestOld.status) {
```

Comment:
at one point, we had planned to log *all* changes in `study_request_changes`.  this was abandoned in favour of the current "status changes only" approach, as this allows us to quickly determine the last time the status was changed to a particular code.

Reply:
(please make a comment to this effect)